.gitlab-ci.yml

```diff
@@ -11,30 +11,37 @@ byod:
   script:
     - . /home/jupyter-user/miniconda/bin/activate
     - pip install --no-cache-dir -r requirements-dev.txt
-    - export GOOGLE_APPLICATION_CREDENTIALS=$TEST_MULE_CREDENTIALS
-    - export GOOGLE_PROJECT=firecloud-cgl
+    - mkdir -p ~/.config/gcloud
+    - cp $TEST_MULE_CREDENTIALS ~/.config/gcloud/application_default_credentials.json
+    - unset GOOGLE_APPLICATION_CREDENTIALS
```

Comment:
Where is `GOOGLE_APPLICATION_CREDENTIALS` set in the first place?

Reply:
Bleh, just when I think I haven't done something blindly.  Really sorry about that.